admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -164,11 +219,34 @@
       onFrameLoaded() {
         this.loadingCombinationForm = false;
         this.submittingCombinationForm = false;
+        const iframeBody = this.$refs.iframe.contentDocument.body;
         this.applyIframeStyling();
+        this.selectedCombinationName = iframeBody.querySelector(
+          ProductMap.combinations.combinationName,
+        ).innerHTML;
+
+        const iframeInputs = iframeBody.querySelectorAll(
+          ProductMap.combinations.editionFormInputs,
+        );
+
+        iframeInputs.forEach((input) => {
+          input.addEventListener('keyup', () => {
```

Comment:
Since you also watch for select elements you should also watch `change` There will be no keyup on select boxes